etcdctlv3/README.md

```diff
@@ -189,9 +189,17 @@ Watch watches events stream on keys or prefixes. The watch command runs until it
 
 - rev -- the revision to start watching. Specifying a revision is useful for observing past events.
 
+#### Input Format
+
+Input is only accepted for interactive mode.
+
+```
+watch [options] <key or prefix>\n
```

Comment:
probably not since when Watch is a sub command, key or prefix is not required (if -i is given). in the interactive input, key or prefix is required.